kodilla-patterns2/src/main/java/com/kodilla/patterns2/observer/homework/Student.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.patterns2.observer.homework;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Student implements Observable {
+    private final String surname;
+    private final List<Observer> observers;
+    private Map<String, String> homeworks;
+    private final KodillaCourses courseName;
+
+    public Student(String surname, KodillaCourses courseName) {
+        this.surname = surname;
+        observers = new ArrayList<>();
+        homeworks = new LinkedHashMap<>();
+        this.courseName = courseName;
```

Comment:
dlaczego surname piszesz z this, a observers juz nie ? Obie wersje sa poprawne, ale nalezy trzymac sie jednek konwencji

Reply:
Zasugerowałem się materiałem Kodilli: https://kodilla.com/static/bootcamp-java/java-20_71.png . Również poprawię.